api/data_refinery_api/views.py

```diff
@@ -1114,6 +1145,15 @@ def _get_intervals(self, objects, range_param, field = 'created_at'):
                       .values('start') \
                       .filter(start__gte=range_to_start_date.get(range_param))
 
+
+STAT_CACHE_REFRESH_INTERVAL_MINUTES = 5
+scheduler = BackgroundScheduler()
+for timeframe in ('day', 'week', 'month', 'year', None):
+    scheduler.add_job(Stats.update_cache, 'interval', [timeframe],
+                      minutes=STAT_CACHE_REFRESH_INTERVAL_MINUTES)
+scheduler.start()
```

Comment:
Is this a good place to instantiate the `BackgroundScheduler`? I'm not sure about it, but I'd imagine these should be in a different file?

Reply:
I'm not sure, since it has to be imported somewhere in order to load.